### PR TITLE
Fix typo in prow controller manager

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

**Review by chases2 — APPROVED** [review]

File: prow/oss/cluster/prow-controller-manager.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ rules:
     resources:
     - leases
     resourceNames:
-    - prow-sinker-leaderlock
+    - prow-controller-manager-leader-lock
```
Comment (chases2) [inline prow/oss/cluster/prow-controller-manager.yaml:68]: Does this change go from `leaderlock` to `leader-lock`?

Comment (chases2) [reply prow/oss/cluster/prow-controller-manager.yaml:68]: /hold

Comment (chaodaiG) [reply prow/oss/cluster/prow-controller-manager.yaml:68]: Yes, it's a hard coded value here: https://github.com/kubernetes/test-infra/blob/8123e82f3ae29afa5e782ca198f9e303c9035673/prow/cmd/prow-controller-manager/main.go#L145

Comment (chaodaiG) [reply prow/oss/cluster/prow-controller-manager.yaml:68]: /unhold

**Pull request merged**